pkg/gameserverallocations/find.go

```diff
@@ -49,9 +49,12 @@ func findGameServerForAllocation(gsa *allocationv1.GameServerAllocation, list []
 			}
 		}
 	case apis.Distributed:
+		// Determine if Priorities are set in the Fleet spec
+		prioritiesSet := len(gsa.Spec.Priorities) > 0
+
 		// randomised looping - make a list of indices, and then randomise them
 		// as we don't want to change the order of the gameserver slice
-		if !runtime.FeatureEnabled(runtime.FeatureCountsAndLists) {
+		if !runtime.FeatureEnabled(runtime.FeatureCountsAndLists) || !prioritiesSet {
```

Comment:
```suggestion
		if !runtime.FeatureEnabled(runtime.FeatureCountsAndLists) || len(gsa.Spec.Priorities) == 0 {
```

Small thing, but probably doesn't need a variable? WDYT @igooch ?

Reply:
No strong feelings either way. It's probably not needed.

Reply:
@ashutosji bump on removing this variable, and just moving the check inline, and then I think this is good to merge.

Reply:
> @ashutosji bump on removing this variable, and just moving the check inline, and then I think this is good to merge.

Thanks! Moved the check inline.